catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ScoreCommand.java

```diff
@@ -0,0 +1,253 @@
+/**
```

Comment:
It seems like there should be an option for this command and the ValidateCommand to run on a metacard that has already been ingested and not just files.